web/styles/settings.css

```diff
@@ -1835,7 +1840,10 @@ label.preferences-radio-choice-label {
             width: 100%;
             border: none;
             /* 48px is the height of settings header and 45px is the height of tab-container */
```

Comment:
I think this comment can be struck, too, since the variables do the talking about what's happening now :)

Reply:
Totally! Thanks